Sources/BLTNItemManager.swift

```diff
@@ -399,6 +400,35 @@ extension BLTNItemManager {
         presentingVC.present(bulletinController, animated: animated, completion: completion)
 
     }
+    
+    /**
+     * Presents the bulletin on top of your application window.
+     *
+     * - parameter application: The application in which to display the bulletin. (normally: UIApplication.shared)
+     * - parameter animated: Whether to animate presentation. Defaults to `true`.
+     * - parameter completion: An optional block to execute after presentation. Default to `nil`.
+     */
+    
+    @objc(showBulletinInApplication:animated:completion:)
+    public func showBulletin(in application: UIApplication,
+                             animated: Bool = true,
+                             completion: (() -> Void)? = nil) {
+        
```

Comment:
Before replacing the `presentingWindow`, you should check if there is one already to avoid potential edge cases.

```suggestion
        assert(presentingWindow == nil, "Attempt to present a Bulletin on top of another Bulletin window. Make sure to dismiss any existing bulletin before calling this method.")
```